package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "cabinet-storage",
   "description": "A simple interface for accessing local and session storage",
-  "version": "0.1.0",
+  "version": "0.2.0",
```

Comment:
probably should be a major version increment because this could be a breaking change when users set falsy values.